application/src/test/java/org/togetherjava/tjbot/commands/basic/PingCommandTest.java

```diff
@@ -1,22 +0,0 @@
-package org.togetherjava.tjbot.commands.basic;
-
-import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
-import org.junit.jupiter.api.Test;
-import org.togetherjava.tjbot.commands.SlashCommand;
-import org.togetherjava.tjbot.jda.JdaTester;
-
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-
-final class PingCommandTest {
-    @Test
-    void pingCommand() {
-        SlashCommand command = new PingCommand();
-        JdaTester jdaTester = new JdaTester();
-
-        SlashCommandEvent event = jdaTester.createSlashCommandEvent(command).build();
-        command.onSlashCommand(event);
-
-        verify(event, times(1)).reply("Pong!");
-    }
-}
```

Comment:
You cant just remove the test... Adjust it to the new code. This PR makes our codebase worse, wont merge something that removes good unit tests... Also, this unit test acted as a showcase of how to unit-test, actively mentioned in the Wiki multiple times, you cant just remove it.